src/gp_game.h

```diff
@@ -59,9 +52,20 @@ typedef enum {
     ACTION_STEP,
     ACTION_TURN_LEFT,
     ACTION_TURN_RIGHT,
+    ACTION_RANDOM,
     ACTION_COUNT,
 } Action;
 
+
+#define STATES_COUNT 7
+#define JEANS_COUNT (STATES_COUNT * ENV_COUNT)
+
+static_assert(
+    JEANS_COUNT % 2 == 0,
+    "Amount of jeans in the chromosome must be an even number for the mating "
+    "process to happen properly.");
```

Comment:
If i'm not mistaken, because of your change to `mate_agents` function, `JEANS_COUNT` can be odd without causing any problem.

Reply:
ah, you're right. i didn't remove it cos tsoding might want to change the mating function back to the original one